src/main/python/lexicon/lexicon_classes.py

```diff
@@ -305,10 +307,25 @@ def unserializerelationmodules(self, serialized_relmodules):
             unserialized[k] = RelationModule(relationx, relationy, bodyparts_dict, contactrel,
                                              xy_crossed, xy_linked, directionslist=directions,
                                              articulators=articulators, timingintervals=timingintervals,
-                                             addedinfo=addedinfo)
+                                             phonlocs=phonlocs, addedinfo=addedinfo)
             unserialized[k].uniqueid = k
         self.relationmodules = unserialized
 
+
+    def unserializehandconfigmodules(self, serialized_hcfgmodules):
```

Comment:
not sure if this is the best way to handle this but it works? Basically running into an issue where it's complaining this property doesn't exist when we try to unserialize it

Reply:
I guess one thing to think about as well is because it sounds like you haven't officially released.. maybe we can just migrate the data structure instead of doing this ?

Reply:
Yeah, this issue has come up repeatedly. I think you're right, that when we're ready to package the first release, we can run an update on the existing .slpaa files and get rid of all the backward-compatibility code (there's a lot) interspersed throughout the project.

Reply:
Sounds good :) 
Although, have we considered how data migration should happen for the end user?  Because I'm sure even after release there will be times where you might want to change the underlying schema.